public/app/features/alerting/unified/components/rule-editor/util.ts

```diff
@@ -64,3 +65,16 @@ export function updateMathExpressionRefs(expression: string, previousRefId: stri
 
   return expression.replace(oldExpression, newExpression);
 }
+
+// some gateways (like Istio) will decode "/" and "\" characters – this will cause 404 errors for any API call
+// that includes these values in the URL (ie. /my/path%2fto/resource -> /my/path/to/resource)
+//
+// see https://istio.io/latest/docs/ops/best-practices/security/#customize-your-system-on-path-normalization
+export function checkForPathSeparator(value: string): ValidateResult {
+  const containsPathSeparator = value.includes('/') || value.includes('\\');
```

Comment:
Question: Checking only for `\\` there. Is `\` safe?

Reply:
Using just the single '\' is an invalid string sequence because `\` is an escape character. For a literal `\` you have to use two of them https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#escape_sequences